components/hab/src/command/pkg/delete.rs

```diff
@@ -0,0 +1,47 @@
+//! Delete a package from Builder.
+//!
+//! # Examples
+//!
+//! ```bash
+//! $ hab pkg delete acme/redis/2.0.7/2112010203120101 stable
```

Comment:
Should this be
```suggestion
//! $ hab pkg delete acme/redis/2.0.7/2112010203120101
```
?

The current command doesn't pass clap parsing:
```
error: Found argument 'stable' which wasn't expected, or isn't valid in this context

USAGE:
    hab pkg delete [OPTIONS] <PKG_IDENT>
```
Since we only are allowing deletion of packages that are not on the stable channel, it seems like we shouldn't be taking a channel arg at all.

Reply:
Yep thats a typo - will fix